### ISSUE TITLE
ServerEndpointConfig connection leak (remains open) if connection is aborted

### ISSUE THREAD

**Issue by szarza** [eclipse-ee4j/tyrus#791]
Websocket endpoint connection remains open even a ServerEndpointConfig.Configurator.modifyHandshake() has been overridden to abort connection. In WSServer.java, if the line 

~~~
@ServerEndpoint(value = "/events/all", configurator = WSServerConfigurator2.class)
~~~

is used, the connection is aborted on the client-side, but not on the server-side, which remains open. Depending on whatever does the server endpoint, that could generate also a memory leak

Attached demonstrator project which includes a file video recording demonstrating that connections are still opened although are closed on the client, showing how to reproduce the issue and including how to find the leak in a heap dump.

[leak-ws.connection leak.tar.gz](https://github.com/eclipse-ee4j/tyrus/files/8877562/leak-ws.connection.leak.tar.gz)

**Comment by jansupol** [comment]
@szarza Thanks for the reproducers. The fixes are in Tyrus 1.19/2.1.0-M4, to appear in maven central as of today.

**Comment by szarza** [comment]
Hi @jansupol:
I'm trying to replace Tyrus libs on Glassfish 6 (version 2.0.3) with the ones of version 2.1.0 but I'm experiencing problems with the compilation: 
 `Failed to execute goal on project tyrus-tests-servlet-embedded-gf: Could not resolve dependencies for project org.glassfish.tyrus.tests.servlet:tyrus-tests-servlet-embedded-gf:war:2.1.0: Could not find artifact org.glassfish.exousia:exousia:jar:2.1.0-M1 in central (https://repo.maven.apache.org/maven2)`
 
 What I should do to have Glassfish 6.2 working with that fixes?
